### DISCUSSION TITLE
How to make the text full screen?

### DISCUSSION THREAD

**Discussion by Just-Math** [3b1b/manim#1425] [Q&A] [answered]
Hello! 
I have a text. I use `ask = TextMobject("""Text on screen""").scale(0.5).to_edge(UL).set_color("#383838")`. 
As a result, the text is visible only half of the screen. How do I make the text appear all over the screen? How to reduce the margin from the edges? Is there any way to make the alignment to the left or to the width.

[screenshot link](https://prnt.sc/10b5s1c)

**Comment by TonyCrane** [comment, marked as answer]
I don't understand you. Maybe `.set_width()` ?

**Reply by Just-Math** [reply]
Thanks for the answer. I have solved this problem. It was necessary to change the sizes of the fields in the tex_template
